api/src/functionalTest/java/uk/gov/hmcts/payment/functional/PBAAccountsFunctionalTest.java

```diff
@@ -155,24 +144,16 @@ private final void performOrganisationCreationWithNoAccounts(final String role,f
         final ValidUser user = idamService.createUserWithSearchScopeForRefData(CMC_CASE_WORKER_GROUP,
             role);
         final String userPUIFinanceManagerToken = user.getAuthorisationToken();
-        System.out.println("The value of the userPUIFinanceManagerToken : " + userPUIFinanceManagerToken);
         final String fileContentsTemplate = readFileContents(INPUT_FILE_PATH + "/" + fileName);
-        System.out.println("The value of the File Contents Before Templating : " + fileContentsTemplate);
         final String fileContents = String.format(fileContentsTemplate,
             generateRandomString(13, true, false),
             generateRandomString(8, true, false),
             user.getEmail());
-        System.out.println("The value of the File Contents After Templating : " + fileContents);
         Response response = postOrganisation(SERVICE_TOKEN_PAYMENT_APP, testProps.getRefDataApiUrl(), fileContents);
-        System.out.println("The value of the Body" + response.getBody().prettyPrint());
         assertThat(response.getStatusCode()).isEqualTo(HttpStatus.CREATED.value());
         final String organisationIdentifier = response.jsonPath().getString("organisationIdentifier");
-        System.out.println(organisationIdentifier);
```

Comment:
Implement HMCTS Logging for the purpose of traceability.....

Reply:
As discussed, will have a new ticket to implement HMCTS Logging across all tests